app/models/volunteer.rb

```diff
@@ -1,4 +1,14 @@
 class Volunteer < ApplicationRecord
+  enum groups: {
```

Comment:
we could use the same values as the keys. The values here will be written to the db.

Reply:
@hantuzun Then we have the issue of how we displayed them in the UI are we OK with just displaying one word descriptions int he dropdown?

Reply:
I usually store `enums` as integers in the database, like so:

https://github.com/hantuzun/pandemic-volunteers/blob/master/app/models/partner.rb#L6

The keys could be full sentences if you want. Then in the UI, you can display the sentences in the drop down:

https://github.com/hantuzun/pandemic-volunteers/blob/master/app/views/partners/new.html.erb#L28

Reply:
Right I usually do the same. @hantuzun suggested using strings instead of integers.

Reply:
Either way is ok with me. If they're strings, we can still use the sentences in the drop down